html/html-quiz.md

```diff
@@ -1936,13 +1936,6 @@ A disabled fieldset is unusable and un-clickable. [Source](https://www.w3schools
 - [x] `<br>`
 - [ ] `<a>`
 
-#### Q112. How to display preformatted text in HTML?
```

Comment:
Moved to the end of the file because of broken numeration (duplicated counter).

Reply:
ok 👍